cif_pow.dic

```diff
@@ -11556,6 +11556,57 @@ save_PD_QPA_EXTERNAL_STD
     _name.object_id               PD_QPA_EXTERNAL_STD
     _category_key.name            '_pd_qpa_external_std.diffractogram_id'
 
+    _description_example.case
+;
+         data_ext_std_diffractogram
+            _pd_diffractogram.id         THE_REFERENCE
+            _pd_diffractogram.instr_id   labmachine
+
+            loop_
+            _pd_phase_mass.phase_id
```

Comment:
`_audit.schema` should be `Custom` if we loop `phase_id`.

Reply:
But there is only one row?

Reply:
Indeed, you are right. I suppose setting up a loop like this might encourage users to think they can add more rows, but I'll leave that up to you to decide.